apps/web/src/components/features/properties/PropertyDetail.tsx

```
@@ -0,0 +1,267 @@
+        {/* Booking Card */}
+        <div className="lg:col-span-1">
+          <Card className="p-6 sticky top-6">
+            <div className="flex items-center justify-between mb-6">
+              <h3 className="text-2xl font-bold">${property.price}</h3>
+              <span className="text-muted-foreground">per night</span>
+            </div>
+
+            <div className="grid grid-cols-2 gap-4 mb-6">
+              <div className="space-y-2">
+                <label className="text-sm font-medium">Check-in</label>
+                <div className="flex items-center border rounded-md p-2 bg-background">
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)